docs/GettingStarted.md

```diff
@@ -0,0 +1,82 @@
+# Getting Started
```

Comment:
Let's add a disclaimer here stressing that they must go over the full docs before deploying anything to prod (since the instructions here are partial).